### DISCUSSION TITLE
How can I format text by programming?

### DISCUSSION THREAD

**Discussion by rudaoshi** [ueberdosis/tiptap#770] [General Questions]
I want to select some text and set it underline or bold by programming. How can I do that?

**Comment by hanspagel** [comment]
Have a look at the default **Bold** node:
https://github.com/ueberdosis/tiptap/blob/89ec3800db4aefa1861fb866d9658a164ec212b3/packages/tiptap-extensions/src/marks/Bold.js

Hope that helps as a starting point!
